test/e2e/storage/vsphere/vsphere_volume_node_poweroff.go

```diff
@@ -178,10 +173,10 @@ func waitForPodToFailover(client clientset.Interface, deployment *extensions.Dep
 	return getNodeForDeployment(client, deployment)
 }
 
-func getNodeForDeployment(client clientset.Interface, deployment *extensions.Deployment) (types.NodeName, error) {
+func getNodeForDeployment(client clientset.Interface, deployment *extensions.Deployment) (string, error) {
```

Comment:
ditto func comment

Reply:
added documentation comment for `getNodeForDeployment` function.